src/OmniSharp.Host/Services/AssemblyLoader.cs

```diff
@@ -60,7 +60,7 @@ public IReadOnlyList<Assembly> LoadAllFrom(string folderPath)
             }
         }
 
-        private Assembly LoadFromPath(string assemblyPath)
+        public Assembly LoadFrom(string assemblyPath)
         {
             Assembly assembly = null;
```

Comment:
I'm with @filipw for consistency lets add a guard `if (string.IsNullOrWhiteSpace(assemblyPath)) return null;`  One less exception to get thrown.